public/javascripts/angularApp.js

```diff
@@ -133,4 +133,11 @@ function($scope, physicians, auth, users){
     //Book An appointment using users service
     users.bookAnAppointment($scope.user, physician, dateTime);
   };
+
+  $scope.goToThisDetails = function(ids,alignToTop){
+    //debugger;
```

Comment:
1. Write a comment above the method explaining what it does & what should be passed as parameters
2. Remove all debuggers